components/generic/w-clickable.vue

```diff
@@ -9,11 +9,12 @@
 </template>
 
 <script>
-import wToggleItem from './w-toggle-item.vue'
-import { computed } from 'vue'
+import { computed } from 'vue';
+import { label as ccLabel } from '@warp-ds/component-classes';
+import wToggleItem from './w-toggle-item.vue';
 
-const clickableClasses = 'absolute inset-0 h-full w-full appearance-none cursor-pointer'
-const labelClasses = 'px-12 py-8 field-label cursor-pointer'
+const clickableClasses = 'absolute inset-0 h-full w-full appearance-none cursor-pointer';
+const labelClasses = `px-12 py-8 ${ccLabel.label} cursor-pointer`
```

Comment:
This need to be moved to the component classes i guess?

Reply:
Yes, it's a helper component, like `Affix`. I should've fixed that while working on the button group component since it was used there, but I thought it was one of components that were listed in the roadmap, so I left it for later. Could we create a task for fixing this @CarolineGard?

Reply:
Created the task here https://trello.com/c/8PELJFoU/224-warpify-clickable-in-vue